mac

```diff
@@ -114,6 +114,7 @@ tap "homebrew/services"
 
 # Unix
 brew "git"
+brew "htop"
 brew "openssl"
 brew "the_silver_searcher"
```

Comment:
I don't think we use this for development?

Reply:
I do from time to time. It can be a handy way to get process IDs and a bit more system info than either `top` or `ps aux`.